Tests/ReactiveSwiftTests/TestError.swift

```diff
@@ -22,16 +21,16 @@ internal extension SignalProducer {
 	/// Halts if an error is emitted in the receiver signal.
 	/// This is useful in tests to be able to just use `startWithNext`
 	/// in cases where we know that an error won't be emitted.
-	func assumeNoErrors() -> SignalProducer<Value, NoError> {
-		return self.lift { $0.assumeNoErrors() }
+	func assumeNevers() -> SignalProducer<Value, Never> {
```

Comment:
This should not be renamed since the re al meaning is now lost.

Reply:
Agreed. The "no error" part of this method name was beyond the `NoError` type.

Reply:
Done. 👍